packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -661,15 +682,22 @@ export class StatsAnalyzer extends EventsScope {
    */
   private compareLastStatsResult() {
     if (this.lastStatsResults !== null && this.meetingMediaStatus) {
-      const getCurrentStatsTotals = (keyPrefix: string, value: string): number =>
+      const getCurrentStatsTotals = (keyPrefix: string, value: string, send: boolean): number =>
```

Comment:
Do we need the `send` param here? Are we not able to tell the direction based on the `keyPrefix`?

Reply:
Yeah that's right, updated.